src/model/format/markdown.ts

```diff
@@ -10,6 +10,7 @@ export class Todo {
     // suffix: '] '
     // body: hello
     private static readonly regexp = /^(?<prefix>((> ?)*)?\s*[\-\*][ ]+\[)(?<check>.)(?<suffix>\]\s+)(?<body>.*)$/;
+    private static readonly checkedStatuses = ['x', '-'];
```

Comment:
To make it customisable as it is, this would have to either
1. be passed from the controller somehow, this will cause the setting to be passed around through every function call.
2. it should access the global settings from the model. This won't work, as the model does not depend on Obsidian, and when importing `{ SETTINGS } from 'settings'` tests will fail

None of these is an acceptable solution, so I decided to leave this as a hardcoded list.

Reply:
I think that's good enough! Thank you!